crates/telio-nurse/src/aggregator.rs

```diff
@@ -15,49 +16,37 @@ use telio_wg::{
     WireGuard,
 };
 
+#[cfg(feature = "mockall")]
+use mockall::automock;
+
 const DAY_DURATION: Duration = Duration::from_secs(60 * 60 * 24);
 
 // Possible Relay connection states - because all of the first 8 bit combinations
 // are reserved for relay states, they need to have the 9th bit on
 #[derive(Copy, Clone, Debug, Eq, Hash, PartialEq)]
 #[repr(u16)]
-pub enum RelayConnectionState {
+pub(crate) enum RelayConnectionState {
     Connecting = 256,
     Connected = 257,
 }
 
-impl From<PeerState> for RelayConnectionState {
-    fn from(value: PeerState) -> Self {
-        if let PeerState::Connected = value {
+impl From<RelayState> for RelayConnectionState {
+    fn from(value: RelayState) -> Self {
+        if let RelayState::Connected = value {
             RelayConnectionState::Connected
         } else {
-            if let PeerState::Disconnected = value {
+            if let RelayState::Disconnected = value {
                 telio_log_warn!("Got disconnected state for Relay server");
```

Comment:
I'm not sure a warning (or any log for that matter) whenever a conversion is done is a good idea.